nd2reader/raw_metadata.py

```diff
@@ -160,7 +161,16 @@ def _parse_z_levels(self):
         Returns:
             list: the z levels, just a sequence from 0 to n.
         """
-        return self._parse_dimension(r""".*?Z\((\d+)\).*?""")
+        z_levels = self._parse_dimension(r""".*?Z\((\d+)\).*?""")
+        if 0 == len(z_levels):
+            z_levels = parse_if_not_none(self.z_data, self._parse_z_coordinates)
+            if isinstance(z_levels, type(None)):
+                z_levels = []
+            else:
+                z_levels = range(len(z_levels))
+            if 0 != z_levels:
```

Comment:
I don't understand this warning. Could you elaborate? Because it seems to me that it is only using the number of "z-coordinates" if `z_levels` is not `None`, right?

Reply:
So, there is definitely an issue, as it should have been `0 != len(z_levels)`, now fixed. The idea I had was to trigger the warning of the fallback behaviour only if no Z-levels are found with the normal approach (of looking for `Z([0-9]+)`), but they are found in the Z-coordinates. I.e., if both approaches yield 0 Z-levels, no warning is needed. What do you think?

Reply:
Ah OK, yes, now it makes sense. However, then I have one small request:
could you remove the `if 0 != len(z_levels):` and put the warning in the `else` clause underneath the `z_levels = range(len(z_levels))` line? That is the only time we have to issue the warning and it cleans the code up a bit. Thank you!

Reply:
Good point! I changed it now.